src/lib.rs

```diff
@@ -422,7 +422,7 @@ impl<'a> DynamicReload<'a> {
     // to read from it directly so this code does some testing first to ensure we
     // can actually read from it (by using metadata which does a stat on the file)
     // If we can't read from it we wait for 100 ms before we try again, if we can't
-    // do it with in 1 sec we give up
+    // do it within 1 sec we give up
```

Comment:
I did not notice that Atom did all those line ending changes... well, If you want to close the pr and just push this fix its fine.